cellprofiler/modules/convertobjectstoimage.py

```diff
@@ -64,15 +62,22 @@ def create_settings(self):
    colors for your objects.
 -  *Binary (black & white):* All object pixels will be assigned 1 and
    all background pixels will be assigned 0, creating a binary image.
--  *Grayscale:* Gives each object a graylevel pixel intensity value
-   corresponding to its number (also called label), so it usually
+-  *Grayscale:* Assigns each object a different number, from 1 to 255
+   (the maximum value that you can put in an 8-bit integer) and numbers all
+   pixels in each object with the object’s number, so it usually
    results in objects on the left side of the image being very dark,
-   progressing toward white on the right side of the image.
--  *uint16:* Assigns each object a different number, from 1 to 65535
-   (the numbers that you can put in a 16-bit integer) and numbers all
-   pixels in each object with the object’s number. This format can be
+   progressing toward white on the right side of the image. This format can be
    written out as a .npy or .tiff file if you want to process the label
    matrix image using another program.
+-  *uint16:* Assigns each object a different number, from 1 to 65535
+   (the maximum value that you can put in a 16-bit integer) and numbers all
+   pixels in each object with the object’s number, so it usually
+   results in objects on the left side of the image being very dark,
+   progressing toward white on the right side of the image (though this can usually 
+   only be seen in a scientific image viewer since standard image viewers only handle
+   8-bit images. This format can be written out as a .npy or .tiff file if you 
```

Comment:
In the style of other documentation, maybe change `This format can be written out as a .npy or .tiff file` to something like `Use **SaveImages** to write the image as a .npy or .tiff file to disk.`